### PR TITLE
React router

### PR DESCRIPTION
**Pull request by vadimshut**
1. Task: https://d14aa3aqnm30ti.cloudfront.net/docs/module-2/home-task/task
2. Screenshot: 
image

 
3. Deploy: -
4. Done 29.06.2022 / deadline 07.07.2022
5. Score: 35/ 35

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,18 +1,69 @@
...
+						<RequireAuth>
+							<CreateCourse
+								authorsList={authors}
+								createNewCourse={createNewCourse}
+							/>
+						</RequireAuth>
+					}
+				/>
+
+				{/*<Route path='*' element={<Navigate to={ROUTES.LOGIN} />} />*/}
```
Comment (ArtemOsadchuck) [inline src/App.jsx]: redundant comment

File: src/components/CourseInfo/CourseInfo.jsx [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+	courseId: PropTypes.string,
+};
+
+CourseInfoUI.defaultProps = {
+	coursesList: [],
+	authorsList: [],
+	courseId: '',
+};
+
+const CourseInfoWrapper = (WrappedComponent) => {
```
Comment (ArtemOsadchuck) [inline src/components/CourseInfo/CourseInfo.jsx]: You can move it to the separate file and rename it like that: CourseInfoContainer(CourseInfoUI)

File: src/App.jsx (hunk 1 above)
Comment (vadimshut) [reply src/App.jsx]: remove comment

File: src/components/CourseInfo/CourseInfo.jsx (hunk 2 above)
Comment (vadimshut) [reply src/components/CourseInfo/CourseInfo.jsx]: move CourseInfoContainer to  separate file

**Pull request merged**